docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+| 5  | Above text is a signature image of stars                                                                                                                                                                                                                 |             |                                                                                    |
+|    | 1.2 Favorite create a tag page 12                                                                                                                                                                                                                        |             |                                                                                    |
+| 6  | Text 1 line "create a tag:" 2 line "create a tag for adding translations to favorites in a well-organized way 3 line placeholder "e.g Lesson1"<br>4 line "You can always manage your tags, renaming, adding, and deleting them in the favorites section" |             |                                                                                    |
+| 7  |                                                                                                                                                                                                                                                          |             |
+| 8  | Texts are in the same format as in layout, text does not go out of screen                                                                                                                                                                                |             |                                                                                    |
+| 9  | Red button "create" is below all text                                                                                                                                                                                                                    |             |                                                                                    |
+| 10 | Above text is a signature image of stars                                                                                                                                                                                                                 |             |                                                                                    |
+| 11 | Validation? For example no # allowed                                                                                                                                                                                                                     |             | perhaps check for sql injections, max number of string [e.t](http://e.tc/).c?      |
```

Comment:
Yes, as we agreed on the last meeting, the maximum allowed length of a tag's name is 100 symbols, it should remove # sign from the user's input if it's the first symbol  + all QA checks that you think we need to do (invalid data, sql and html injections, etc.)